src/actions/actionStyles.ts

```diff
@@ -43,7 +43,7 @@ export const actionPasteStyles: Action = {
             roughness: pastedElement?.roughness,
           };
           if (isTextElement(newElement)) {
-            newElement.font = pastedElement?.font;
+            newElement.font = pastedElement?.font || "20px Virgil";
```

Comment:
`"20px Virgil"` feels like it could be a const.. used already in few places.

Reply:
Yea sure, I thought about making it a const

Reply:
I'll make a PR tonight(EU time)

Reply:
Also would be nice to wait for someone to approve a PR before merging next time :)